dawn/src/dawn/IIR/StencilInstantiation.cpp

```diff
@@ -48,22 +48,24 @@ namespace iir {
 //===------------------------------------------------------------------------------------------===//
 
 StencilInstantiation::StencilInstantiation(
-    sir::GlobalVariableMap const& globalVariables,
+    const ast::GridType gridType, sir::GlobalVariableMap const& globalVariables,
```

Comment:
as we decided not to decide on where to put the const, I would just adapt to the surrounding code. Here: put it to the right.

Reply:
done, thx for the hint